packages/stress/src/mode/chat/schema/executeCommand.ts

```diff
@@ -0,0 +1,80 @@
+import { StressClient } from '../../../utils/stressClient'
+import { ChatConfig } from '../../common/types'
+import { Command } from './testPlanSchema'
+import { getLogger } from '../../../utils/logger'
+import { joinSpace } from './commands/joinSpaceCommand'
+import { check } from '@river-build/dlog'
+import { mintMemberships } from './commands/mintMembershipCommand'
+import { sendChannelMessage } from './commands/sendChannelMessageCommand'
+import { expectChannelMessages } from './commands/expectChannelMessageCommand'
+
+export async function executeCommand(
+    command: Command,
+    commandId: string, // For logging / error reporting
+    chatConfig: ChatConfig,
+    clients: StressClient[],
+) {
+    const logger = getLogger('stress:run', { function: 'executeCommand' })
+    logger.info({ command }, '--------------- executing command ---------------')
+
+    const range = (start: number, end: number) =>
+        Array.from({ length: end - start }, (v, k) => k + start)
```

Comment:
endIndex inclusive? i.e. it should be end-start + 1 ?

Reply:
Non-inclusive! clientEndIndex is not included in the indices of clients running on this process.